src/server.c

```diff
@@ -2769,6 +2769,7 @@ void initServer(void) {
      * no explicit limit in the user provided configuration we set a limit
      * at 3 GB using maxmemory with 'noeviction' policy'. This avoids
      * useless crashes of the instance for out of memory. */
+
```

Comment:
Invalid line addition.

Reply:
Removed th eline and updated diff, Thanks!